tests/client.yaml

```diff
@@ -20,9 +20,11 @@ volumes:
 
 services:
   validator-0:
-    image: sawtooth-validator:$ISOLATION_ID
+    image: sawtooth-pbft-test:$ISOLATION_ID
```

Comment:
Can you add more information to the commit message to explain why these changes belong in the same commit?

Reply:
Addressed